docs/guide/catalog/product.md

```diff
@@ -43,12 +43,23 @@ silversolutions_eshop:
             - my_custom_product_type
 ```
 
-Additionally, to enable buying the product in the shop,
-add the following parameter for your custom Content Type's identifier:
+The custom Content Type must mirror the structure of the built-in `ses_product` Content Type,
+including having Fields with the same identifiers,
```

Comment:
Is it "the structure that includes the Fields with the same identifiers", or is it "by including/having the Fields with the same IDs", or would it be just "The custom Content Type must include all the Fields with the same identifiers as the built-in `ses_product` Content Type, ..."?